integrations/github-app/syncs/commits.ts

```
@@ -0,0 +1,98 @@
+    let cursor: string | undefined;
+    let commitCount = 0;
+
+    while (hasNextPage) {
+        if (shouldAbortSync(startTime)) {
+            await nango.log('Aborting sync due to 20 hours time limit', { level: 'warn' });
+            break;
+        }
+        if (cursor) {
+            variables['cursor'] = cursor;
+        } else {
+            delete variables['cursor'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)